tabbycat/draw/models.py

```diff
@@ -183,27 +187,31 @@ def debateteams_ordered(self):
         for side in self.round.tournament.sides:
             yield self.get_dt(side)
 
-    aff_team = _team_property('aff_team')
-    neg_team = _team_property('neg_team')
-    bye_team = _team_property('bye_team')
-    og_team = _team_property('og_team')
-    oo_team = _team_property('oo_team')
-    cg_team = _team_property('cg_team')
-    co_team = _team_property('co_team')
-    aff_dt = _team_property('aff_dt')
-    neg_dt = _team_property('neg_dt')
-    bye_dt = _team_property('bye_dt')
-    og_dt = _team_property('og_dt')
-    oo_dt = _team_property('oo_dt')
-    cg_dt = _team_property('cg_dt')
-    co_dt = _team_property('co_dt')
-
-    def get_team(self, side):
-        return getattr(self, '%s_team' % side)
-
-    def get_dt(self, side):
+    aff_team = _team_property('0_team')
+    neg_team = _team_property('1_team')
+    bye_team = _team_property('-1_team')
+    og_team = _team_property('0_team')
+    oo_team = _team_property('1_team')
+    cg_team = _team_property('2_team')
+    co_team = _team_property('3_team')
+    aff_dt = _team_property('0_dt')
+    neg_dt = _team_property('1_dt')
+    bye_dt = _team_property('-1_dt')
+    og_dt = _team_property('0_dt')
+    oo_dt = _team_property('1_dt')
+    cg_dt = _team_property('2_dt')
+    co_dt = _team_property('3_dt')
```

Comment:
I think originally I wrote `_team_property` to work on the assumption that `attr` (the argument to `_team_property`) would be a valid attribute name. That's obviously no longer true and I guess it doesn't need to be. So if you like you're welcome to change the naming convention to e.g. `team_0`, `dt_0`, _etc._, if you'd find that more intuitive.

Reply:
I'd be more inclined to delete these and use indexes in lists of `teams` and `debateteams`.

Reply:
Fine with me, I think I originally mostly did this whole `_team_property` thing as a backwards compatibility thing, and I also didn't want to go and change every instance of `.aff_team` and `.neg_team` to an index because the code sometimes read better with it phrased that away. This makes less sense when generalising to arbitrary numbers of teams.